solutions/setup-dev-environment/local-up.sh

```
@@ -26,7 +27,13 @@ $(echo ${joincmd} --force-internal-endpoint-lookup --wait --context ${c1ctx} | s
 echo -e "Join cluster2 to hub\n"
 $(echo ${joincmd} --force-internal-endpoint-lookup --wait --context ${c2ctx} | sed "s/<cluster_name>/$c2/g")
 
-echo -e "Accept join of cluster1 and cluster2\n"
-clusteradm accept --context ${hubctx} --clusters ${c1},${c2} --wait
+echo -e "Join hub to itself as ${local_cluster}\n"
+$(echo ${joincmd} --force-internal-endpoint-lookup --wait --context ${hubctx} | sed "s/<cluster_name>/${local_cluster}/g")
```

Shellcheck (0.11.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)